src/reconcile/rec_row.c

```diff
@@ -526,7 +526,12 @@ __wti_rec_row_int(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_PAGE *page)
 
         /* Build key cell. Truncate any 0th key, internal pages don't need 0th keys. */
         __wt_ref_key(page, ref, &p, &size);
-        if (r->cell_zero)
```

Comment:
Is this really needed?

Reply:
Discussed offline, we do not wish to alter keys when building deltas.